src/app/Library/CrudPanel/CrudFilter.php

```diff
@@ -146,7 +146,7 @@ public function getViewWithNamespace()
     public function getNamespacedViewWithFallbacks()
     {
         $type = $this->type;
-        $namespaces = config('backpack.crud.view_namespaces.filters');
+        $namespaces = array_merge(config('backpack.crud.view_namespaces.filters'), $this->crud()->get('viewNamespaces')['filters'] ?? []);
```

Comment:
The fact that I see this `array_merge()` twice makes me thing "this should probably be a function".